src/frontend/komponenter/Fagsak/DataValidator.tsx

```diff
@@ -0,0 +1,36 @@
+const harFyltInnFødselsnummer = (filled: String): boolean => {
```

Comment:
The file should end with .ts since it is only a typescript file and not a react component.